crimsobot/utils/image.py

```diff
@@ -603,36 +603,41 @@ async def process_image(ctx: Context, image: Optional[str], effect: str, arg: Op
                 await crimsogames.win(ctx.guild.me, cost)
                 new_bal = await crimsogames.check_balance(ctx.author)
 
-    embed = c.crimbed(
-        title='PLS TO HOLD...',
-        descr='\n'.join([
-            f'Processing GIF for **{ctx.author}**...',
-            f'{img.width} \u2A09 {img.height} pixels · {img.n_frames} frames',
-        ]),
-        footer=f'GIF cost: \u20A2{cost:.2f} · Your balance: \u20A2{bal:.2f} ➡️ \u20A2{new_bal:.2f}',
-        color_name='yellow',
-        thumb_name='wizard',
-    )
-
-    msg = await ctx.send(embed=embed)
+    is_gif = img.format == 'GIF'
```

Comment:
i'd move this up and use the `getattr(img, 'is_animated', False)` check instead of relying on the image being a gif... just because it's a gif doesn't mean it's animated 😉

Reply:
THIS is a good point, I will do this